addon-resizer/nanny/estimator_test.go

```diff
@@ -239,3 +239,23 @@ func TestEstimateResources(t *testing.T) {
 		verifyResources(t, "requests", got.Requests, want.Limits)
 	}
 }
+
+func TestComputeResourceOverheadValueString(t *testing.T) {
+	testCases := []struct {
+		numNodes         uint64
+		resource         Resource
+		expectedOverhead string
+	}{
+		{1, Resource{Base: resource.MustParse("1"), ExtraPerNode: resource.MustParse("1"), Name: "cpu"}, "1.000000"},
```

Comment:
This function computes per node overhead, so Base values are not needed in test cases.

Reply:
Done.